CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 This project uses a version scheme based on the Cadenza main version in the format x.x.y, where x.x is the Cadenza main version and y a functional change or bugfix.
 
 ## Unreleased
+### Fixed
+- Removed duplicate geometry section in `sandbox.html#editGeometry`
+- Horizontal scrollbar in `sandbox.html` left sidebar now appears sticky.
```

Comment:
A Horizontal scrollbar is rather painful to use. Why do we want it here?

Reply:
We currently have it due to long explaination texts, and this is a improvement to the current state imo.

Reply:
But the sidebar is fixed width. We should be able to craft the contents in a way that does not require a horizontal scrollbar. For example texts should wrap lines. We can have a look if necessary.